websocketOutportDriver/data/interface.go

```diff
@@ -0,0 +1,10 @@
+package data
+
+import "io"
+
+// WSConn defines what a sender shall do
+type WSConn interface {
```

Comment:
maybe add also IsInterfaceNil

Reply:
I cannot do that because this interface is over `websocket.Conn`